common/persistence/sql/sqlplugin/sqlite/cluster_metadata.go

```diff
@@ -264,6 +264,5 @@ func (mdb *db) PruneClusterMembership(
 		templatePruneStaleClusterMembership,
 		constMembershipPartition,
 		mdb.converter.ToSQLiteDateTime(filter.PruneRecordsBefore),
-		filter.MaxRecordsAffected,
```

Comment:
Shall we create an issue or add a TODO for removing this field from the filter?

Reply:
I do think we need to revisit all of these sqlplugins

this copy/pasta issue needs to be prevented with some code-gen.

Reply:
I should probably just do that with this PR.

@yiminc thoughts?

Reply:
Removed the field for the filter only for sql plugin.